Makefile

```diff
@@ -649,6 +649,11 @@ fmt-go:
 	cd server/log-collector && \
 		make fmt
 
+.PHONY: vale-docs
+vale-docs: ## Run vale check for docs and sorts ignore.txt file
```

Comment:
Why not add this to the `lint-docs` rule?

Reply:
@moranbental it's already exist and used within linting ci

Reply:
@moranbental done